src/v/rpc/test/BUILD

```diff
@@ -101,6 +118,37 @@ redpanda_cc_btest(
     ],
 )
 
+redpanda_cc_btest(
+    name = "rpc_gen_cycling_test",
+    timeout = "short",
+    srcs = [
+        "rpc_gen_cycling_test.cc",
+    ],
+    cpu = 1,
+    data = [
+        ":cert",
+        ":cert_ca",
+        ":other_cert",
+        ":other_cert_ca",
+    ],
+    deps = [
+        ":cycling_rpc",
+        ":echo_rpc",
+        ":echo_v2_rpc",
+        ":rpc_integration_fixture",
+        "//src/v/bytes:random",
+        "//src/v/model",
+        "//src/v/random:generators",
+        "//src/v/rpc",
+        "//src/v/test_utils:fixture",
+        "//src/v/test_utils:seastar_boost",
+        "@boost//:test",
+        "@seastar",
+        "@seastar//:testing",
+    ],
+    #exclusive?
```

Comment:
yes for now we need it 😄

Reply:
oh... i  forgot that there :sweat_smile: